modular_skyrat/modules/modular_weapons/code/medcells.dm

```diff
@@ -508,6 +508,47 @@
 		usr.visible_message(span_notice("[usr] deactivates \the [src]."), span_notice("You deactivate \the [src]."))
 		qdel(src)
 
+//Oppressive Force Relocation
+/obj/item/ammo_casing/energy/medical/utility/sec_remover
+	projectile_type = /obj/projectile/energy/medical/utility/sec_remover
+	select_name = "sec-b-gone"
+	select_color = "#850000"
+
+/obj/projectile/energy/medical/utility/sec_remover
+	name = "bluespace transportation field"
+
+/obj/projectile/energy/medical/utility/sec_remover/on_hit(mob/living/target)
+	. = ..()
+
+	var/mob/living/carbon/human/officer = target
+
+	if(!istype(get_area(target), /area/medical)) //We don't want this to work everywhere, do we?
+		return FALSE
+
+	var/target_access = officer.wear_id.GetAccess() //Stores the access of the target within a variable
+	if(!(ACCESS_BRIG in target_access))
+		return FALSE
+
+	teleport_effect(officer.loc)
+
+	var/list/turf_list = list()
+
+	for(var/turf/turf_in_area in get_area_turfs(/area/security/brig))
+		if(!turf_in_area.is_blocked_turf())
+			turf_list += turf_in_area
```

Comment:
I feel like we could afford to make that static instead, for better performance, ngl
```suggestion
	var/static/list/turf_list

	if(!turf_list)
		turf_list = list()
		for(var/turf/turf_in_area in get_area_turfs(/area/security/brig))
			if(!turf_in_area.is_blocked_turf())
				turf_list += turf_in_area
```